engine/buildkit/llbdefinition.go

```diff
@@ -147,6 +153,8 @@ func (dag *OpDAG) walk(f func(*OpDAG) error, memo map[*OpDAG]struct{}) error {
 	return nil
 }
 
+var SkipInputs = fmt.Errorf("skip inputs") //nolint:stylecheck // Err prefix isn't convention for Walk control errors
```

Comment:
Would it be beneficial to ignore just the style check rule that's flagging the `Err prefix` here instead of the whole style check linter? I'm not sure what the convention is for dagger but I have done this kind of think with Java lint tools in private codebases before.